### PR TITLE
[Reviewer: Matt] Allow the license and project URL to be overridden

### PR DESCRIPTION
**Pull request by AlexHockey**
This PR enhances the RPM specfile includes to allow the RPM's license and URL to be overriden.

### CODE REVIEW

File: cw-rpm.spec.inc [hunk 1]
```diff
@@ -1,7 +1,11 @@
+# Default the License and URL if they are not already specified.
+%{!?LICENSE_VALUE: %define LICENSE_VALUE "GPLv3+ with exceptions" }
```
Comment (mirw) [inline cw-rpm.spec.inc]: Why `_VALUE`?  No strong objection - it just felt a bit odd.

**Comment by mirw** [comment]
Looks good.  1 comment above.

File: cw-rpm.spec.inc (hunk 1 above)
Comment (AlexHockey) [inline cw-rpm.spec.inc]: I wanted a consistent naming convention. So `X_VALUE` is the value of the `X` field.

Comment (mirw) [inline cw-rpm.spec.inc]: For consistency, shouldn't this be `RPM_LICENSE` and `RPM_URL`, then?

**Pull request merged**